index.html

```diff
@@ -186,86 +188,88 @@ <h3>US cities may be under-reporting their emissions... by up to 145%</h3>
 
     <!-- Funding -->
     <section class="funding">
-      <div class="container">
-        <h2>Latest funding rounds on Index.co <a href="#"><i class="fas fa-chevron-right"></i></a></h2>
-        <div class="row-latest">
-          <div class="item-h">
-            <div class="item-heading">
-              <div class="item-h-left">
-                <img src="assets/h1.jpg" alt="" class='img-h'>
-              </div>
-              <div class="item-h-right">
-                <h4 class='h4-red'>Podimo_Global</h4>
-                <div class="p-linen">
-                  <p>€11.2M</p>
-                  <span class='span-normal'>VENTURE CAPITAL</span>
-                </div>
-              </div>
-            </div>
-            <div class="item-text">
-              <p>Podimo, the podcast and short form audio subscription service, picks up €11.2M in new funding on
-                TechCrunch</p>
-            </div>
 
-          </div>
-          <div class="item-h">
-            <div class="item-heading">
-              <div class="item-h-left">
-                <img src="assets/h2.webp" alt="" class='img-h'>
-              </div>
-              <div class="item-h-right">
-                <h4 class='h4-red'>Klook</h4>
-                <div class="p-linen">
-                  <p>€11.2M</p>
-                  <span class='span-normal'>VENTURE CAPITAL</span>
-                </div>
-              </div>
+      <a href="#">
+        <h2>Latest funding rounds on Index.co <i class="fas fa-chevron-right"></i></h2>
+      </a>
+      <div class="row-latest">
+        <div class="item-h">
+          <div class="item-heading">
+            <div class="item-h-left">
```

Comment:
## `Latest Funding` - Screen Change
- [ ] The Change in the `Latest Funding` section requires that the articles get to span the entire page. In the clone page, 👇 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/MarkoNS1990/the_next_web/milestone-2/index.html)|
|:--:|:--:|
|![latest_funding_small_screen](https://user-images.githubusercontent.com/22345961/109027370-c1681b80-76d1-11eb-977b-915029e3c02b.png)|![latest_funding_small_screen_clone](https://user-images.githubusercontent.com/22345961/109027377-c5943900-76d1-11eb-9547-6e58d8a2a0cc.png)|